meetings/200325-board-meeting.md

```diff
@@ -104,3 +104,128 @@
 ### CarpentryConnect 2021
 - instead of ResBaz, a CarpentryConnect event could be held
 - ideas/comments/experiences welcome
+
+
+# Minutes 
+
+*purpose of the meeting*: handover.
+
+### Presents:
+
+- Annika Rockenberger
+- Anne Schad Bergsaker
+- Lex Nederbragt
+- Naoe Tatara
+- Tobias Busch
+- Viviane Girardin
+- Anne Fouilloux
+- Elin Stangeland
+- Sabry Razick
+
+### Regrets:
+
+- Andrea A. Gasparini
+- Live Rasmussen
+
+
+It was the first online zoom meeting for the board & interim board.
+
+Before starting the meeting, tips for using Zoom have been exchanged within the participants; for instance, on how to get the grid view (as we are few people).
+
+## Presentation of the new board and farewell/Thank you to the interim board
+
+Everyone gave a short presentation (interim board and then new board).
+
+
+# Introduction to Carpentry@UiO and Governance Documents
+
+- The interim board explained the rational behind these documents.
+- Interim board has finished writing governance documents however, some errors need to be corrected:
+     * Carpentry@UiO is not registered as an official organization. It is more like a "club".
+- Abide to the Carpentry Code of Conduct
+- No mention of Florida charter while the UiO governance documents were inspired by it.
+
+
+## Officer roles
+
+The new board needs to be organized and appoint officers (secretatry, etc.). 
+
+- the board secretary is in charge of the meeting minutes but does not have to write minutes all the time; Secretary is in charge of making sure there is a notetaker and that minutes are pushed to github.
+
+- The board believes there is a need for a Communication officer.
+
+- 2 members of the current board will be appointed for 1 year and 2 members for 2 years
+      --> how to do it?  
+      --> will be discussed later to decide how to do with the re-election of 2 members.
+
+The board agreed on the fact that the chair and vice-chair should have some experience. However, this will be re-discussed in a separate meeting.
+
```

Comment:
Should we add a link to the officer roles for the Carpentries Executive Council? https://github.com/carpentries/executive-council-info/blob/master/process/roles_responsibilities.md#officers